guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -34,3 +34,12 @@ ifdef::satellite,orcharhino[]
 This migration takes time, so a pre-migration runs automatically after the upgrade to {ProjectVersion} to reduce future upgrade downtime.
 While the pre-migration is running, {ProjectServer} is fully functional but uses more hardware resources.
 endif::[]
+
+. Optional: Delete the `/var/lib/pgsql/data-old/` directory by entering the following command:
```

Comment:
```suggestion
. Optional: Delete the `/var/lib/pgsql/data-old/` directory:
```

Reply:
The other steps in the module were explicit about that part, so I went by that "boilerplate". Personally I definitely prefer the more minimalist version, though :-)